[INA-7367] Update Incident Management docs for Confluence Postmortems (#28986)
DataDog/documentation: content/en/service_management/incident_management/incident_settings/templates.md

## Postmortems

> Postmortem templates are dynamic, reusable templates used to create a [Datadog Notebook][2] or [Confluence Page][3] that is automatically populated with incident information after an incident has been resolved. Postmortem templates leverage template variables, such as `{{incident.severity}}`, to dynamically inject the corresponding value from the incident that the postmortem is being created for. 

Reviewer: ```suggestion
Postmortem templates are dynamic, reusable templates used to create a [Datadog Notebook][2] or [Confluence page][3] that is automatically populated with incident information after an incident has been resolved. Postmortem templates leverage template variables, such as `{{incident.severity}}`, to dynamically inject the corresponding value from the incident that the postmortem is being created for. 
```

To create a postmortem template:

1. Click the **+ New Postmortem Template** button
2. Give the template a name

> 3. Select either a Datadog Notebook or a Confluence Page to be created at generation
> 4. Write the template's content (available template variables are listed to the right of the textbox)
> 5. (Optional) Set the template as the default 
> 6. Click **Save**

Reviewer: ```suggestion
3. Choose either a Datadog Notebook or a Confluence page to be generated.
4. Write the template's content using template variables (available to the right of the textbox)
5. (Optional) Set the template as the default 
6. Click **Save**
```
